tests/testthat/test-preparedata-tidytable.R

```diff
@@ -374,22 +374,7 @@ tb.relpct <- structure(c(5.00326180684867, 7.285650634158, 8.40518598530936,
                 Minimum = 999L, Maximum = 999L, Range = FALSE, Total = 1000L,
                 Missing = 0L, EffectiveSampleSize = 999L, EffectiveSampleSizeProportion = 100,
                 FilteredProportion = 0), questiontypes = c("Ranking", "PickOne"
-            ), span = list(rows = structure(list(c("Overall, I feel confident and optimistic that I could prepare for, cope with and recover from an epidemic event in the future",
-            "I feel able to protect my home and family from a potential epidemic",
-            "Even outside times of risk of illness, I regularly do things like clean and disinfect my house and its contents",
-            "In reality, the risk of illness from an epidemic is pretty low in my area",
-            "To be honest, preparing or planning for a potential future epidemic is not a top priority for me",
-            "I dont really know what I should do to prepare or plan for a future epidemic",
-            "I feel I can deal with whatever comes", "During a future epidemic I'd do whatever I could to protect myself and my family",
-            "Rather than relying on authorities, I would make my own decisions about how to respond during a future epidemic",
-            "I feel confident I have the financial resources to recover if my workplace and therefore my income was impacted by a future epidemic",
-            "I feel confident we would receive timely and accurate information from authorities about what to do",
-            "I could rely on my neighbours or friends to help me if I was affected by a future epidemic",
-            "There is good community spirit in my area, people get involved and help out others"
-            )), class = "data.frame", .Names = "", row.names = c(NA, 13L)),
-                columns = structure(list(c("Segment 1", "Segment 2", "Segment 3",
-                "Segment 4", "NET")), class = "data.frame", .Names = "", row.names = c(NA,
-                5L))), name = "table.AS.RANKING.Q17.How.strongly.do.you.agree.or.disagree.with.by.Segments.using.AS.RANKING.Q17", questions = c("[AS RANKING] Q17. How strongly do you agree or disagree with ... ?",
+            ), name = "table.AS.RANKING.Q17.How.strongly.do.you.agree.or.disagree.with.by.Segments.using.AS.RANKING.Q17", questions = c("[AS RANKING] Q17. How strongly do you agree or disagree with ... ?",
```

Comment:
This test no longer has a span?

Reply:
Yes, this an old test and doesn't have a QTable class. In this case it should drop the span because none of the verbs subscripting is applied and its safer just to drop it. Its a bit lossy but at least it won't trigger a hardship error.